src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -41,6 +41,13 @@ public IEnumerable<Assembly> RetrievePluginAssemblies(IEnumerable<Assembly> defa
             return assemblies;
         }
 
+        public bool IsTargetWithinFileSizeLimit(string path, int maxFileSize)
+        {
+            long fileSize = FileSystem.GetFileSize(path) / 1024;
+
+            return (maxFileSize == -1 || fileSize < maxFileSize);
```

Comment:
>fileSize < maxFileSize [](http://example.com/codeflow?start=41&length=22)

I think a few cases are missing:
maxfile = -1 vs  -1/0/1/big
maxfile = file = 10 (a reasonable number)

Reply:
Updated test cases and added some explicit follow up confirming this method separately.